From 3e6e4421a8bedcd9cf374025e3fac1412036bc95 Mon Sep 17 00:00:00 2001
From: Owen W. Taylor <otaylor@fishsoup.net>
Date: Tue, 30 Aug 2011 18:17:19 +0000
Subject: Fix a stray brace

A recent commit introduced an unclosed brace for an if statement
---
diff --git a/plugins/xsettings/gsd-xsettings-gtk.c b/plugins/xsettings/gsd-xsettings-gtk.c
index 0ce4db7..c2a043c 100644
--- a/plugins/xsettings/gsd-xsettings-gtk.c
+++ b/plugins/xsettings/gsd-xsettings-gtk.c
@@ -76,7 +76,7 @@ cond_setting_changed (GSettings       *settings,
 
         enabled = g_settings_get_boolean (settings, key);
         if (enabled != FALSE) {
-                if (gtk->priv->dir_modules == NULL) {
+                if (gtk->priv->dir_modules == NULL)
                         gtk->priv->dir_modules = g_hash_table_new_full (g_str_hash, g_str_equal, g_free, NULL);
                 g_hash_table_insert (gtk->priv->dir_modules, g_strdup (module_name), NULL);
         } else if (gtk->priv->dir_modules != NULL) {
--
cgit v0.9.0.2

